src/analysis/retail/monk/mistweaver/modules/spells/AncientTeachings.tsx

```diff
@@ -177,6 +210,34 @@ class AncientTeachingsoftheMonastery extends Analyzer {
     };
   }
 
+  getTooltip(spellId: number, secondarySourceId?: number) {
+    return (
+      <>
+        From your {this.damageSpellsCount.get(spellId) || 0} <SpellLink id={spellId} />{' '}
```

Comment:
```suggestion
        From your {this.damageSpellsCount.get(spellId) || 0} <SpellLink id={spellId} /> casts{' '}
```

Reply:
Adding 'casts' doesn't make sense for bok_totm which is why I left it out